be/src/testing/testing.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, MethodNotAllowedException } from '@nestjs/common';
+import { ItemService } from '../item/item.service';
+import { CreateItemInput } from 'src/item/dto/create-item.input';
+import { CategoryService } from 'src/category/category.service';
+import { CreateCategoryInput } from 'src/category/dto/create-category.input';
+import { CreateNoteInput } from 'src/note/dto/create-note.input';
+import { NoteService } from 'src/note/note.service';
+import { UserService } from 'src/user/service/user.service';
+import { CreateUserDTO } from 'src/user/dto/user.create.dto';
+import { Role } from 'src/user/roles/user.roles';
+import * as faker from 'faker';
+
+@Injectable()
+export class TestingService {
+  constructor(
+    private readonly categoryRepo: CategoryService,
+    private readonly itemRepository: ItemService,
+    private readonly noteRepo: NoteService,
+    private readonly userRepo: UserService,
+  ) {
+    if (process.env.ADD_FAKE_DATA === 'TRUE') this.AddFakeData();
+    console.log('add fake data ', process.env.ADD_FAKE_DATA);
+  }
+
+  AddFakeData() {
+    if (process.env.ADD_FAKE_DATA !== 'TRUE') {
+      throw new MethodNotAllowedException(`ADD_FAKE_DATA not set to TRUE`);
+    }
+    for (let i = 0; i < 3; i++) {
+      const item: CreateItemInput = { regCode: faker.datatype.number() };
+      this.itemRepository.createItem(item);
+
+      const category: CreateCategoryInput = {
+        name: faker.name.findName(),
+        bundelable: faker.datatype.boolean(),
+        parentCategoryId: null,
+      };
+      this.categoryRepo.create(category);
+
+      const note: CreateNoteInput = { content: faker.lorem.text(), itemId: 1 };
```

Comment:
Also add some notes to other items as well?

Reply:
I don't really know how i can easily add other ids. Should i get the number of ids from the db and then generate a number between the number of ids and 1? Or would there be some other way?

Reply:
The `createItem` function you are using returns the item it had created. You can then store the id's of the items and add notes to them

Reply:
done in -> [98ab51](https://github.com/TPT-Loane/TPT-Loane/pull/120/commits/98ab5169b3b4faf4959945cbe16e8b67b06e2ed4)